packages/cardano-services/src/WsServer/server.ts

```diff
@@ -220,9 +220,17 @@ export class CardanoWsServer extends WsProvider {
RkKGFkZHJlc3Nlc1tpXSk7XG59XG5jb25zdCBhZGRyZXNzZXMgPSBbLi4uYWRkcmVzc2VzU2V0XTsiLCJkZXBlbmRlbmNpZXMiOltdLCJuYW1lIjoiRXZlbiBiZXR0ZXIifSx7ImlkIjoiTFcyTGJEaVBBT3AtMlNFZVVoSFJFIiwiY29kZSI6ImNvbnN0IHsgd3NzIH0gPSBEQVRBO1xuXG5jb25zdCBhZGRyZXNzZXNNYXAgPSB7fTtcbmZvciAoY29uc3Qgd3Mgb2Ygd3NzLmNsaWVudHMpXG4gIGZvciAoY29uc3QgYWRkcmVzcyBvZiB3cy5hZGRyZXNzZXMpXG4gICAgYWRkcmVzc2VzTWFwW2FkZHJlc3NdID0gdHJ1ZTtcbmNvbnN0IGFkZHJlc3NlcyA9IE9iamVjdC5rZXlzKGFkZHJlc3Nlc01hcCk7IiwiZGVwZW5kZW5jaWVzIjpbXSwibmFtZSI6Ik9iamVjdCAxIn0seyJpZCI6Im9LMm5IQ3JJTzg2bFl2TmwyMGprbyIsImNvZGUiOiJjb25zdCB7IHdzcyB9ID0gREFUQTtcblxuY29uc3QgYWRkcmVzc2VzTWFwID0ge307XG5cbmZvciAoY29uc3Qgd3Mgb2Ygd3NzLmNsaWVudHMpIHtcbiAgY29uc3QgYWRkcmVzc2VzID0gd3MuYWRkcmVzc2VzO1xuICBmb3IgKGxldCBhTGVuID0gYWRkcmVzc2VzLmxlbmd0aDsgYUxlbi0tOylcbiAgICBhZGRyZXNzZXNNYXBbYWRkcmVzc2VzW2FMZW5dXSA9IHRydWU7XG59XG4gIFxuY29uc3QgYWRkcmVzc2VzID0gT2JqZWN0LmtleXMoYWRkcmVzc2VzTWFwKTsiLCJkZXBlbmRlbmNpZXMiOltdLCJuYW1lIjoiT2JqZWN0IDIifSx7ImlkIjoiT0p0TklnRVBvb3BoaDNoVHdhQktHIiwiY29kZSI6ImNvbnN0IHsgd3NzIH0gPSBEQVRBO1xuXG5jb25zdCBhZGRyZXNzZXNNYXAgPSB7fTtcblxuZm9yIChjb25zdCB3cyBvZiB3c3MuY2xpZW50cylcbiAgZm9yIChjb25zdCBhZGRyZXNzIG9mIHdzLmFkZHJlc3NlcylcbiAgICBhZGRyZXNzZXNNYXBbYWRkcmVzc10gPSB0cnVlO1xuICBcbmNvbnN0IGFkZHJlc3NlcyA9IE9iamVjdC5rZXlzKGFkZHJlc3Nlc01hcCk7IiwiZGVwZW5kZW5jaWVzIjpbXSwibmFtZSI6Ik9iamVjdCAzIn1dLCJjb25maWciOnsibmFtZSI6IkJhc2ljIGV4YW1wbGUiLCJwYXJhbGxlbCI6dHJ1ZSwiZ2xvYmFsVGVzdENvbmZpZyI6eyJkZXBlbmRlbmNpZXMiOlt7InVybCI6Imh0dHBzOi8vY2RuLmpzZGVsaXZyLm5ldC9ucG0vQGZha2VyLWpzL2Zha2VyQDguMC4yLytlc20iLCJuYW1lIjoiRkFLRVIiLCJlc20iOnRydWV9XX0sImRhdGFDb2RlIjoiY29uc3QgeyB1dWlkIH0gPSBGQUtFUi5mYWtlci5zdHJpbmc7XG5jb25zdCBhZGRyZXNzZXMgPSBbXTtcbmNvbnN0IGNsaWVudHMgPSBbXTtcblxubGV0IGFJZHggPSAwO1xuXG4vLyBTaW11bGF0ZSAxMDAwIGNvbm5lY3RlZCBjbGllbnRzIHdpdGggMTAwIHN1YnNjcmliZWQgYWRkcmVzc2VzXG5mb3IobGV0IGNJZHg7IGNJZHggPCAxMDAwMDsgY0lkeCsrKSB7XG4gIGNvbnN0IHdzID0geyBhZGRyZXNzZXM6IFtdIH07XG5cbiAgLy8gU2ltdWxhdGUgMSUgb2YgdXNlcnMgaGF2ZSB0aGUgc2FtZSB3YWxsZXQgY29ubmVjdGVkIGZyb21cbiAgLy8gdHdvIGRpc3RpbmN0IExhY2UgaW5zdGFuY2VzXG4gIGlmKGNJZHggJiYgTWF0aC5yYW5kb20oKSA8IDAuMDEpXG4gICAgY2xpZW50cy5wdXNoKGNsaWVudHNbW01hdGguZmxvb3IoTWF0aC5yYW5kb20oKSAqIGNsaWVudHMubGVndGgpXV0pO1xuICBlbHNlXG4gICAgZm9yKGxldCBpID0gMDsgaSA8IDEwMDsgKytpKSB7XG4gICAgICAvLyBTaW11bGF0ZSAwLjElIG9mIGFkZHJlc3NlcyBpcyBzdWJzY3JpYmVkIGJ5IGRpc3RpbmN0IHVzZXJzXG4gICAgICAvLyBpLmUuIHNoYXJlZCB3YWxsZXRzXG4gICAgICBpZihjSWR4ICYmIE1hdGgucmFuZG9tKCkgPCAwLjAwMSlcbiAgICAgICAgd3MuYWRkcmVzc2VzLnB1c2goYWRkcmVzc2VzW01hdGguZmxvb3IoTWF0aC5yYW5kb20oKSAqIGFkZHJlc3Nlcy5sZWd0aCldKTtcbiAgICAgIGVsc2Uge1xuICAgICAgICAvLyBBcGFydCBmcm9tIHRoZSBmaXhlZCBwcmVmaXhcbiAgICAgICAgLy8gYWRkcmVzc2VzIGFuZCB1dWlkcyBhcmUgbG9uZyBzZXF1ZW5jZXMgb2YgcmFuZG9tIGNoYXJhY3RlcnNcbiAgICAgICAgY29uc3QgYWRkcmVzcyA9IFwiYWRkcl9cIiArIHV1aWQoKTtcblxuICAgICAgICBhZGRyZXNzZXNbYUlkeCsrXSA9IGFkZHJlc3M7XG4gICAgICAgIHdzLmFkZHJlc3Nlcy5wdXNoKGFkZHJlc3MpO1xuICAgICAgfVxuXG4gICAgICBjbGllbnRzLnB1c2god3MpO1xuICAgIH1cbn1cblxucmV0dXJuIHsgd3NzOiB7IGNsaWVudHM6IG5ldyBTZXQoY2xpZW50cykgfSB9OyJ9fQ
```

Comment:
Oof....
```suggestion
          // Benchmarks: https://t.ly/L4Z9s
```

Reply:
They do not specify the lifetime, hopefully 'long enough'? Can you think of any other alternatives?

Reply:
It's a link in a comment... we already spent on it much more time than how much it deserves.
I have only three options:

1. remove the link
2. leave it as is
3. apply your suggestion accepting the risk we will lode the info in a couple of months

Reply:
Leave it as it is. Also lets create a cloudflare account in the background, it should be useful for cases like this